pages/_app.tsx

```diff
@@ -38,7 +39,7 @@ const ERROR_SCREEN_STYLES: ChakraProps = {
   justifyContent: 'center',
   width: 'fit-content',
   maxW: '800px',
-  margin: '0 auto',
+  margin: { base: '0 auto', lg: '0 auto' },
```

Comment:
?

Reply:
That is for more specicity and overriding underlying component styles.